### PR TITLE
[Protocol Plugin] Replace bootstrap with fabric styles

### PR DESCRIPTION
**Pull request by abuccts**
Replace bootstrap with fabric styles.

Asset size: `plugin.js 1.25MB -> 652KB`.

### CODE REVIEW

File: contrib/submit-protocol/src/App/ProtocolForm.tsx [hunk 1]
```diff
@@ -137,58 +276,50 @@ export default class ProtocolForm extends React.Component<IProtocolProps, IProto
...
+            <Stack className={styles.item}>
+              <Toggle
+                label="Job Parameters"
+                checked={this.state.showParameters}
+                onChange={this.toggleParameters}
+                inlineLabel={true}
+              />
+              {this.renderParameters()}
+            </Stack>
+            <Stack horizontal={true} horizontalAlign={"end"} className={styles.footer}>
```
Comment (Gerhut) [inline contrib/submit-protocol/src/App/ProtocolForm.tsx]:
```suggestion
            <Stack horizontal={true} horizontalAlign="end" className={styles.footer}>
```

File: contrib/submit-protocol/src/App/ProtocolForm.tsx [hunk 2]
```diff
@@ -115,6 +215,45 @@ export default class ProtocolForm extends React.Component<IProtocolProps, IProto
       />
     );
 
+    const uploadOptions = [
+      {
+        key: "local",
+        text: "",
+        onRenderField: (props: any, render: any) => {
```
Comment (Gerhut) [inline contrib/submit-protocol/src/App/ProtocolForm.tsx]: Would this be type-annotated?

File: contrib/submit-protocol/src/App/ProtocolForm.tsx [hunk 3]
```diff
@@ -87,20 +184,23 @@ export default class ProtocolForm extends React.Component<IProtocolProps, IProto
...
-              </div>
-              <div className={classNames(bootstrapStyles.modalBody, bootstrapStyles.row)}>
-                <Spinner size={SpinnerSize.large} />
-              </div>
-            </div>
-          </div>
-        </div>
+        <Stack>
+          <Stack gap={20} padding={20} horizontalAlign="center" className={styles.form}>
+            <Stack horizontal={true} horizontalAlign={"center"} className={styles.header}>
```
Comment (Gerhut) [inline contrib/submit-protocol/src/App/ProtocolForm.tsx]:
```suggestion
            <Stack horizontal={true} horizontalAlign="center" className={styles.header}>
```

**Comment by abuccts** [comment]
Here's the new form with Office Fabric styles:

image

image

**Pull request merged**